src/common/__tests__/kube-helpers.test.ts

```diff
@@ -0,0 +1,179 @@
+const logger = {
+  silly: jest.fn(),
+  debug: jest.fn(),
+  log: jest.fn(),
+  info: jest.fn(),
+  error: jest.fn(),
+  crit: jest.fn(),
+};
+
+jest.mock("winston", () => ({
+  format: {
+    colorize: jest.fn(),
+    combine: jest.fn(),
+    simple: jest.fn(),
+    label: jest.fn(),
+    timestamp: jest.fn(),
+    printf: jest.fn()
+  },
+  createLogger: jest.fn().mockReturnValue(logger),
+  transports: {
+    Console: jest.fn(),
+    File: jest.fn(),
+  }
+}))
+
+interface kubeconfig {
+  apiVersion: string,
+  clusters: [{
+    name: string,
+    cluster: {
+      server: string
+    }
+  }],
+  contexts: [{
+    context: {
+      cluster: string,
+      user: string,
+    },
+    name: string
+  }],
+  users: [{
+    name: string
+  }],
+  kind: string,
+  'current-context': string,
+  preferences: {}
+}
+
+import { loadConfig } from "../kube-helpers"
+import { KubeConfig } from "@kubernetes/client-node";
+
+console.log("") // fix bug
```

Comment:
```suggestion
console = new Console(process.stdout, process.stderr) // fix bug
```

Reply:
Actually I don't think I need it as this test set is not using mockFS so have removed it.